### DISCUSSION TITLE
dicom print

### DISCUSSION THREAD

**Discussion by Georgeqhh** [nroduit/Weasis#326] [Ideas]
DicomPrint class provides two methods to print dicom,
printImage(BufferedImage image)
printImage(ExportLayout<? extends ImageElement> layout)
In my java program,I want to print dicom by customizing the layout not the ExportLayout but the easy layout param...

**Comment by nroduit** [comment]
You are free to implement as you wish. The implementation was made here to send in DICOM a 1/1 format because some printers do not support it correctly or have proprietary functions.

**Reply by Georgeqhh** [reply]
It is too hard for me to implement by myself,if possible,add a method that just need a pojo, and then convert to the ExportLayout...then it will be very easy for java program to use the dicom print function

**Comment by nroduit** [comment]
See also https://github.com/nroduit/Weasis/issues/238#issuecomment-1022939645
